src/test/java/io/odh/test/e2e/standard/MultipartFormDataBodyPublisher.java

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright Skodjob authors.
+ * Copyright (c) 2022 yusuke suzuki
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.odh.test.e2e.standard;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.math.BigInteger;
+import java.net.http.HttpRequest.BodyPublisher;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.nio.ByteBuffer;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Formatter;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+import java.util.StringJoiner;
+import java.util.concurrent.Flow.Subscriber;
+import java.util.function.Supplier;
+
+/**
+ * multipart/form-data BodyPublisher.
+ */
+public class MultipartFormDataBodyPublisher implements BodyPublisher {
```

Comment:
I don't see any reason why non test class is in test package -> move it to main please